src/form-question-types.rst

```diff
@@ -1655,33 +1655,86 @@ In the parameters column, write ``max-pixels=`` followed by the desired maximum
 Audio widget
 ----------------
 
-Records audio from an external app.
+type
+  :tc:`audio`
+appearance
+  :tc:`none`
 
-.. note::
+Records audio using the device's microphone.
 
-  Some Android devices provide a default application for audio recording.
-  Others do not,
-  and the user will need to install an audio recording app.
+.. image:: /img/form-widgets/audio-start.*
+  :alt: The Audio form widget as displayed in the ODK Collect App on an Android phone. The question text is "What does it sound like?" There are two buttons: Record Sound and Choose Sound.
 
-  There are many apps available for this, including:
+.. rubric:: XLSForm
 
-  - `Axet Audio Recorder (open source) <https://play.google.com/store/apps/details?id=com.github.axet.audiorecorder>`_
-  - `RecForge II <https://play.google.com/store/apps/details?id=dje073.android.modernrecforge>`_
+.. csv-table:: survey
+  :header: type, name, label
 
-  Any app that responds to
-  `android.provider.MediaStore.Audio.Media.RECORD_SOUND_ACTION`
-  will be compatible.
+  audio, sound_like, What does it sound like?
 
 
-.. image:: /img/form-widgets/audio-start.*
-  :alt: The Audio form widget as displayed in the ODK Collect App on an Android phone. The question text is "Please record your name." There are three buttons: Record Sound, Choose Sound and Play Sound. The "Play Sound" button is disabled.
+Customizing audio quality
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The quality of audio recordings can be customized using the ``quality`` parameter. The available quality values are:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Value
+     - Extension
+     - Encoding
+     - Bit rate
+     - Sample rate
+     - File size
+   * - normal
+     - .m4a
+     - AAC
+     - 64kbps
+     - 32kHz
+     - ~30MB/hour
+   * - voice-only
+     - .amr
+     - AMR
+     - 12.2kbps
+     - 8kHz
+     - ~5MB/hour
+
+.. tip::
+
+  We'd recommend only using ``voice-only`` for one-on-one interviews in a quiet place as otherwise there might be too much detail loss. If in doubt, it's a good idea to test the different qualities out to see which one fits your use case best.
+
+  If it's a possibility that an individual question could need different qualities depending on context you can use :ref:`relevance <relevants>` to switch between them.
+
+If no ``quality`` is specified, ``normal`` is used.
+
 
 .. rubric:: XLSForm
 
+In the parameters column, write ``quality=`` followed by the desired value.
+
 .. csv-table:: survey
-  :header: type, name, label
+ :header: type, name, label, parameters
+
+ audio,voice_only_audio,Voice audio,quality=voice-only
+
+Recording with an external app
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+  Before Collect 1.29 audio questions would always use an external app. 
```

Comment:
Is this the most useful place for this warning? Maybe it's more the `Customizing audio quality` section that needs a warning saying it's only available in 1.29+.

Reply:
Yeah that sounds better to me!